instrumentation/servlet/servlet-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/snippet/SnippetInjectingResponseWrapper.java

```diff
@@ -50,9 +60,10 @@ public boolean containsHeader(String name) {
 
   @Override
   public void setHeader(String name, String value) {
-    if (isContentTypeTextHtml() && "Content-Length".equalsIgnoreCase(name)) {
+    // checking content-type is just an optimization to avoid unnecessary parsing
```

Comment:
I'd drop this comment also in the other similar places below also

Reply:
didn't find the other places that also have the similar comment.